openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml

```diff
@@ -44,21 +44,31 @@
   narrative:
     - key: a
       text: |
-        A complete control response is planned. Engineering progress can be
+        Machine Config Operators (MCO) allow for periodic update to
+        its baseline configuration using MachineConfig objects.
+        for more information:
+        https://github.com/openshift/machine-config-operator
```

Comment:
Awesome job contextualizing this control for OpenShift!

Are there official docs that could be used instead of GitHub?